index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+</head>
+<body>
+  <div class="header">
+    <a href="#" class="welcome miss-desktop">Welcome</a>
+    <nav class="nav">
+      <a href="#" class="menu miss-desktop"><i class="material-symbols-rounded">menu</i></a>
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>  
+  </div>
+  <main>
+    <header class="headline">
+      <div class="item">
+        <h1 class="intro">Hey there, I'm Harriet</h1>
+        <h2 class="intro green">I'm a software developer</h2>
+      </div>
+      <p class="item"> 
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="socials item">
+        <li><i class="fab fa-brands fa-twitter"></i></li>
+        <li><i class="fab fab-brands fa-linkedin-in"></i></li>
+        <li><i class="fab fa-brands fa-github"></i></li>
+        <li><i class="fa fa-regular fa-hand-peace"></i></li>
+      </ul>
```

Comment:
- [ ] Well done using the `ul/li` tags as suggested by the reviewer 🥇🥇 I noticed that because of using `ul/li`, the list items have a `list-style-type`, which was not present previously. You don't need to change `ul/li`, you did absolutely right job there 😃 However, please make sure to use `list-style-type: none` for list items to match the figma template 💪

### _Your design previously 🔽_
![image](https://user-images.githubusercontent.com/106119105/219650831-1381bc45-aef6-4846-ac8f-2538ec54a054.png)

### _Your design now 🔽_
![image](https://user-images.githubusercontent.com/106119105/219650905-2de9c911-612b-4c28-8744-bdff588a4473.png)

### _Figma design 🔽_
![image](https://user-images.githubusercontent.com/106119105/219650960-5cae3c42-4c77-4d38-a34b-caf2fd0d5a9a.png)